lib/test-utils.ts

```diff
@@ -152,14 +185,29 @@ export const newContext = async (
 	};
 
 	const flushAll = async (session: string) => {
-		try {
-			while (true) {
-				await flush(session);
+		await new Promise((resolve) => {
+			setTimeout(resolve, 10);
+		});
```

Comment:
Why the mandatory wait? Always forcing a wait without a logical path in the code that leads to it is usually a red flag that something is wrong.

Reply:
Here the function is "yielding" so that other functions ( like consuming from the queue and adding to the array that is going to process later ) can run.
I will double-check that this is necessary, though.

Reply:
@ramirogm Did you find out why this new mandatory wait is now necessary?

Reply:
They're not needed anymore after the updates I did to `transformers.spec`, so I removed them

Reply:
Rollback. I had to put it back because 17 tests were failing